### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by erciccione**
- add initial docs/CONTRIBUTING.md
- update CODEOWNERS.md, README.md and bounties.md
- fix some issues found by codacy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,33 +37,36 @@ Haveno is a community-driven project. For it to be succesful it's fundamental to
...
+
+## Bounties
 
-See the [FAQ in the wiki](https://github.com/haveno-dex/haveno/wiki/FAQ).
+To incentivize development and reward contributors we adopt a simple bounty system. Contributors may be awarded bounties after completing a task (resolving an issue). [More details in the docs](docs/bounties.md).
 
 ## Running a local Haveno test network
 
 1. Download [Monero CLI](https://www.getmonero.org/downloads/) for your system and sync Monero stagenet: `./monerod --stagenet --rpc-login superuser:abctesting123`, or alternatively, [set up a local Monero stagenet network](#running-a-local-monero-stagenet-network) (recommended)
-3. Download and install [Bitcoin-Qt](https://bitcoin.org/en/download)
```
Comment (woodser) [inline README.md:53]: This is #2 on master.  Is your branch up-to-date?

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+## Contributing to Haveno
```
Comment (woodser) [inline docs/CONTRIBUTING.md:1]: Why is this file all uppercase and the others are lowercase?  Should be consistent.

File: README.md [hunk 3]
```diff
@@ -22,18 +24,10 @@ The project is divided between multiple repositories:
...
 - **[haveno-design](https://github.com/haveno-dex/haveno-design)** - Temporary (for now empty) repository for design discussions.
 - **[haveno-meta](https://github.com/haveno-dex/haveno-meta)** - For project-wide discussions and proposals.
-- **[haveno-website](https://github.com/haveno-dex/haveno-website)** - The repository of the future website.
+- **[haveno-website](https://github.com/haveno-dex/haveno-website)** - The repository of the website.
 
 Currently, efforts are focused in developing the core repository ('haveno'). If you wish to help, take a look at the [issue tracker](https://github.com/haveno-dex/haveno/issues) and the [Kanban boards (projects)](https://github.com/haveno-dex/haveno/projects). We run a bounty program to incentivize development, the issues elegible for a reward in XMR [are labelled '💰bounty'](https://github.com/haveno-dex/haveno/labels/%F0%9F%92%B0bounty).
 
-## Trade protocol
-
-See [docs/trade-protocol.md](docs/trade-protocol.md)
```
Comment (woodser) [inline README.md:31]: Burying the only link to the trade protocol in the FAQ?

I figured we should collect links to our ./docs somewhere.

File: README.md (hunk 1 above)
Comment (erciccione) [reply README.md:53]: It is. My last commit is 8756d8feca8d55a70470e92d38d4cd1c7021b226. Weird.

File: README.md (hunk 3 above)
Comment (erciccione) [reply README.md:31]: I removed this by mistake, as i introduced it in this position with #75. Fixed now

File: docs/CONTRIBUTING.md (hunk 2 above)
Comment (erciccione) [reply docs/CONTRIBUTING.md:1]: uppercase files are meant to interact with GitHub's UI: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

File: README.md (hunk 1 above)
Comment (erciccione) [reply README.md:53]: So, apparently GitHub is autocorrecting the list, but it's wrong in the readme. This PR restores the correct order.

**Comment by erciccione** [comment]
rebased (#78 created conflicts)

File: docs/bounties.md [hunk 4]
```diff
@@ -1,7 +1,10 @@
 ## Bounties
 
-We use bounties to incentivize development. To receive a bounty, you agree to these conditions:
+We use bounties to incentivize development and reward contributors. All issues available for a bounty are listed [on the Kanban board](https://github.com/orgs/haveno-dex/projects/2). It's possible to list on each repository the issues with a bounty on them, by searching issues with the '💰bounty' label.
 
+To receive a bounty, you agree to these conditions:
+
+- Your changes must follow the [styling guidelines](contributing.md).
```
Comment (woodser) [inline docs/bounties.md:7]: Is this link correct since it's in ./docs?

**Comment by woodser** [comment]
Sorry, some comments were pending in an incomplete review.

File: docs/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+Thanks for wishing to help! Here there are some guidelines and information about the development process. We suggest you to join the matrix room `#haveno-dev` (relayed on Freenode) and have a chat with the devs, so that we can help get you started.
+
+Issues are tracked on GitHub. We use [a label system](https://github.com/haveno-dex/haveno/issues/50) and GitHub's [project boards](https://github.com/haveno-dex/haveno/projects) to simplify development. Make sure to take a look at those and to follow the priorities suggested.
+
+### General guidelines
```
[the remark this answers is not in the document]
Comment (erciccione) [reply docs/CONTRIBUTING.md:7]: That's meant to be a child of "Contributing to Haveno".

File: README.md [hunk 6]
```diff
@@ -43,33 +41,36 @@ Haveno is a community-driven project. For it to be succesful it's fundamental to
 
 Temporary email: havenodex@protonmail.com
 
-## FAQ
+### Styling guidelines
```
[the remark this answers is not in the document]
Comment (erciccione) [reply README.md:44]: It's meant to be a child of "Keep in touch and help out!"

File: docs/bounties.md (hunk 4 above)
Comment (erciccione) [reply docs/bounties.md:7]: This file is in `/docs` too, it's a relative link. I guess could be changed to `/docs/CONTRIBUTING.md` if wanted (the capitalization needs to be fixed regardless).

**Pull request merged**